RELEASE.md

```diff
@@ -22,68 +22,72 @@ This page describes the release process and the currently planned schedule for u
 
 Maintaining the release branches for older minor releases happens on a best effort basis.
 
-### 1. Prepare your release
-
-At the start of a new major or minor release cycle create the corresponding release branch based on the main branch. 
-For example if we're releasing `0.6.0` and the previous stable release is `0.5.0` we need to create a `release-0.6` branch. 
-Note that all releases are handled in protected release branches, see the above `Branch management and versioning` section. 
-Release candidates and patch releases for any given major or minor release happen in the same `release-<major>.<minor>` branch. 
-Do not create `release-<version>` for patch or release candidate releases.
-
-Note that it need to update the image version of `fluentbit-operator` to the `release-<version>`.
-
-Changes for a patch release or release candidate should be merged into the previously mentioned release branch via pull request.
-
-Bump the version in the `VERSION` file. Do this in a proper PR pointing to the release branch as this gives others the opportunity to chime in on the release in general and on the addition to the changelog in particular. 
-For a release candidate, append something like `-rc.0` to the version (with the corresponding changes to the tag name, the release name etc.).
+## Prepare your release
 
-For release candidates still update `CHANGELOG.md`, but when you cut the final release later, merge all the changes from the pre-releases into the one final update.
+For a new major or minor release, work from the `main` branch. For a patch release, work in the branch of the minor release you want to patch (e.g. `release-0.1` if you're releasing `v0.1.1`).
 
-Entries in the `CHANGELOG.md` are meant to be in this order:
+Add an entry for the new version to the `CHANGELOG.md` file. Entries in the `CHANGELOG.md` should be in this order:
 
 * `[CHANGE]`
 * `[FEATURE]`
 * `[ENHANCEMENT]`
 * `[BUGFIX]`
 
-### 2. Draft the new release
+Create a PR for the changes to be reviewed.
 
-Tag the new release via the following commands:
+## Publish the new release
 
+For new minor and major releases, create the `release-<major>.<minor>` branch starting at the PR merge commit.
+From now on, all work happens on the `release-<major>.<minor>` branch.
+
+Bump the version in the `VERSION` file in the root of the repository.
+
+Regenerate setup.yaml based on latest code and then commit the changed bundle.yaml to the `release-<major>.<minor>` branch:
 ```bash
-$ tag="v$(< VERSION)"
-$ git tag -s "${tag}" -m "${tag}"
-$ git push origin "${tag}"
+make manifests
+git add ./
+git commit -s -m "regenerate setup.yaml"
+git push
 ```
 
-Optionally, you can use this handy `.gitconfig` alias.
+Manually build and push the container image:
 
-```ini
-[alias]
-  tag-release = "!f() { tag=v${1:-$(cat VERSION)} ; git tag -s ${tag} -m ${tag} && git push origin ${tag}; }; f"
+```bash
+make build-op-amd64 -e FO_IMG=<image of fluent operator>
```

Comment:
We've CI pipeline already to build images automatically, do we still need manually build and push?

Reply:
No need. As a supplement here. Shall we remove it?

Reply:
OK.